linuxbrew-build.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+ABS_PATH_LIMIT=85
+export HOMEBREW_NO_AUTO_UPDATE=1
+
+[[ -x ./bin/brew ]] || (echo "This script should be run inside Linuxbrew directory."; exit 1)
+
+cd $(realpath .)
+BREW_HOME="$PWD"
+
+LEN=${#BREW_HOME}
+[[ $LEN -eq $ABS_PATH_LIMIT ]] || (echo "Linuxbrew absolute path should be exactly $ABS_PATH_LIMIT \
+ bytes, but actual length is $LEN bytes: $BREW_HOME"; exit 1)
+
+export HOMEBREW_ARCH=ivybridge
+
+openssl_formula=./Library/Taps/homebrew/homebrew-core/Formula/openssl.rb
+openssl_orig=./Library/Taps/homebrew/homebrew-core/Formula/openssl.rb.orig
+
+if [[ ! -e "$openssl_orig" ]]; then
+  ./bin/brew info openssl >/dev/null
+  cp "$openssl_formula" "$openssl_orig"
+fi
+
+cp "$openssl_orig" "$openssl_formula"
+  cat <<EOF | patch -n $openssl_formula
```

Comment:
Nit: this line should not be indented.
```suggestion
cat <<EOF | patch -n "$openssl_formula"
```
Also `$openssl_formula` should be quoted.